src/datePicker/index.tsx

```diff
@@ -90,10 +90,12 @@ const StyledDatePicker = styled(DatePicker)`
     color: ${color.fadedText};
     cursor: default;
   }
-  & .DayPicker-Day--selected span {
+  & .DayPicker-Day--selected span,
+  & .DayPicker-Day:focus span {
```

Comment:
Making the focus state visible

Reply:
Hmm not really I don't control this part, we use react-day-picker

Reply:
let's use at least the > selector to make the rule a bit less lax.